src/broker/cas_execute.c

```diff
@@ -1247,7 +1247,9 @@ ux_execute (T_SRV_HANDLE * srv_handle, char flag, int max_col_size, int max_row,
     }
 #endif /* !LIBCAS_FOR_JSP && !CAS_FOR_ORACLE && !CAS_FOR_MYSQL */
 
+  hm_set_current_srv_handle (srv_handle->id);
```

Comment:
Shouldn't it be added to ux_execute_batch(), ux_execute_array() ?

Reply:
a server handler is not assigned for `ux_execute_batch()`.